docs/home/contribute/localization.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Localizing Kubernetes Documentation
+---
+
+We're happy to add localizations of Kubernetes documentation to the website!
+
+Localizations must meet the following requirements for _workflow_ (how to localize) and _output_ (what to localize).
+
+## Workflow  
+
+- All localization work must happen within a Kubernetes organization repository specifically dedicated to localization.
+
+    For example: https://github.com/kubernetes/kubernetes-docs-cn
+
+    To open a localization repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
```

Comment:
This is not so clear about who to @ exactly on the slack channel...

Reply:
@xiangpengzhao I intentionally omitted a specific name because the lead role for SIG Docs may change over time and I wanted to future-proof the page. That said, Slack may not be the best venue for an initial contact&mdash;@dankohn, any thoughts here?

Reply:
I might suggest a message on the SIG-docs channel, but what you have is fine. You might also mention that you can find the SIG-docs leads at https://github.com/kubernetes/community/blob/master/sig-list.md